.github/actions/start-console/action.yml

```diff
@@ -0,0 +1,81 @@
+name: Start console
+description: Start console using docker compose.
+inputs:
+  ui_image:
+    description: image uri for the ui (ie. ghcr.io/<namespace>/<image-name>:<tag>)
+    type: string
+    required: false
+    default: ""
+  server_image:
+    description: image uri for the server (ie. ghcr.io/<namespace>/<image-name>:<tag>)
+    type: string
+    required: false
+    default: ""
+  server_db_image:
+    description: image uri for server-postgres (ie. ghcr.io/<namespace>/<image-name>:<tag>)
+    type: string
+    required: false
+    default: ""
+  playwright_version:
+    description: version of the playwright image to run
+    type: string
+    required: false
+    default: ""
+outputs:
+  server_port:
+    description: Port where the server is running
+    value: ${{ steps.set-output.outputs.server_port }}
+  ui_port:
+    description: Port where the UI is running
+    value: ${{ steps.set-output.outputs.ui_port }}
+  playwright_port:
+    description: Port where the UI is running
```

Comment:
I think this should say "Port where the Playwright server is running", to avoid confusion with `ui_port`